src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -38,13 +39,17 @@ export const postSubmit = (req: Request, res: Response) => {
   return res.redirect(config.UPDATE_CHECK_YOUR_ANSWERS_URL);
 };
 
-// With validation in place we will only have 3 choices
-const getNextPage = (beneficialOwnerTypeChoices: BeneficialOwnerTypeChoice): string => {
-  if (beneficialOwnerTypeChoices === BeneficialOwnerTypeChoice.government) {
-    return config.UPDATE_BENEFICIAL_OWNER_GOV_URL;
-  } else if (beneficialOwnerTypeChoices === BeneficialOwnerTypeChoice.otherLegal) {
-    return config.UPDATE_BENEFICIAL_OWNER_OTHER_URL;
-  } else {
-    return config.UPDATE_BENEFICIAL_OWNER_INDIVIDUAL_URL;
+const getNextPage = (beneficialOwnerTypeChoices: BeneficialOwnerTypeChoice | ManagingOfficerTypeChoice): string => {
+  switch (beneficialOwnerTypeChoices) {
+      case BeneficialOwnerTypeChoice.government:
+        return config.UPDATE_BENEFICIAL_OWNER_GOV_URL;
+      case BeneficialOwnerTypeChoice.otherLegal:
+        return config.UPDATE_BENEFICIAL_OWNER_OTHER_URL;
+      case BeneficialOwnerTypeChoice.individual:
+        return config.UPDATE_BENEFICIAL_OWNER_INDIVIDUAL_URL;
+      case ManagingOfficerTypeChoice.individual:
+        return config.UPDATE_MANAGING_OFFICER_INDIVIDUAL_URL;
+      default:
```

Comment:
Should fail as default or be IndividualBO.

Reply:
Failure scenario won't hit the controller let alone failing on default, empty selection will fail on validation check and not call controller. Individual BO is already on line 49